### DISCUSSION TITLE
how to auto clear space after some time to store new logs and remove old logs

### DISCUSSION THREAD

**Discussion by usmanaa10** [Security-Onion-Solutions/securityonion#7476] [General] [answered]
Hi Friends,
               I have small storage and I can only store 15 days logs so what should I do so it can automatically remove old logs and make free space to store next-day logs instead of manually clearing space e.g so-elastic-clear etc.
Kindly guide me properly with the configuration step and the theory behind it.
I am using security onion  2.3.100

Thanks

**Comment by tomjones1977** [comment, marked as answer]
`Curator` is a tool that runs automatically to clear space for new logs by removing older logs.  If it is not doing so in time you may need to adjust the settings and thresholds for when it removes old data.

https://docs.securityonion.net/en/2.3/curator.html#curator
